src/genie/libs/parser/iosxe/tests/ShowSwitchVirtual/cli/equal/golden_output_virtual_expected.py

```diff
@@ -0,0 +1,12 @@
+expected_output = {    
+    "mode": "Virtual Switch",
+    "domain_number": 123,
+    "switches": {
+        "1" : {
+            "role": "Active"
+        },
+        "2" : {
+            "role": "Standby"
+        }
+    }
+}
```

Comment:
My apologies, I didn't notice this before. It looks like this output isn't being parsed correctly; it's missing some information from the device output. It should look something more like this.
```suggestion
expected_output = {
    'vss_member_id': {
        1: {
            'role': "VSS Active",
            'mode': "Virtual Switch",
            'domain_number': 123,
            'local_switch': {
                'switch_number': 1,
                'operational_role': 'Virtual Switch Active'
            },
            'peer_switch': {
                'switch_number': 2,
                'operational_role': 'Virtual Switch Standby'
            }
        },
        2: {
            'role': "VSS Standby",
            'mode': "Virtual Switch",
            'domain_number': 123,
            'local_switch': {
                'switch_number': 2,
                'operational_role': 'Virtual Switch Standby'
            },
            'peer_switch': {
                'switch_number': 1,
                'operational_role': 'Virtual Switch Active'
            }
        }
    }
}
```

Reply:
Updated with a new commit to get recommended returned dictionary.